sdk/core/azure-core/test/ut/CMakeLists.txt

```diff
@@ -64,4 +60,6 @@ target_link_libraries(azure-core-test PRIVATE azure-core gtest gmock)
 # gtest_discover_tests will scan the test from azure-core-test and call add_test
 # for each test to ctest. This enables `ctest -r` to run specific tests directly.
 gtest_discover_tests(azure-core-test
-     TEST_PREFIX azure-core.)
+     TEST_PREFIX azure-core.
+     NO_PRETTY_TYPES
+     NO_PRETTY_VALUES)
```

Comment:
Let me try out these settings locally in a bit to see how they behave, before merging.